pages/repo/raw.js

```diff
@@ -231,9 +234,12 @@ export default compose(
               lineWrapping: true,
               smartIndent: false,
               viewportMargin: Infinity,
-              foldGutter: true,
-              gutters: ['CodeMirror-linenumbers', 'CodeMirror-foldgutter'],
-              foldOptions: process.browser && {
+              foldGutter: foldCode,
+              gutters: [
+                'CodeMirror-linenumbers',
+                foldCode && 'CodeMirror-foldgutter'
+              ].filter(Boolean),
+              foldOptions: foldCode && {
```

Comment:
IMO leave the `process.browser` in there in case some one changes the `foldCode` logic to be true during SSR
```suggestion
              foldOptions: process.browser && foldCode && {
```

Reply:
Yes I had it at first but then I thought this just adds noise to the code – maybe it's valuable documentation to keep it in though.